ding/league/starcraft_player.py

```diff
@@ -220,15 +220,15 @@ def is_trained_enough(self) -> bool:
         return super().is_trained_enough(select_fn=lambda p: isinstance(p, HistoricalPlayer))
 
     # override
-    def mutate(self, info) -> Optional[str]:
+    def mutate(self, info) -> Union[str, None]:
```

Comment:
How does it decide when to mutate LE and ME?

Reply:
In current setting, we implement mutate mechanism following AlphaStar pseudo code.